core/src/test/java/org/apache/druid/data/input/impl/JsonInputFormatTest.java

```diff
@@ -69,7 +69,7 @@ public void testEquals()
               new ObjectMapper(),
               new ObjectMapper()
               )
-              .withIgnoredFields("objectMapper")
+              .withIgnoredFields("objectMapper", "lineSplittable")
```

Comment:
`lineSplittable` shouldn't be ignored, but included in `equals()` and `hashCode()`.

Reply:
The new property `lineSplittable` is not declared as `final` and `EqualsVerifier` here will report a warning that `equals depends on mutable field`, so the new commit calls `suppress` method on `EqualsVerifier` to suppress this warning.

Reply:
Hmm, good point. Yeah, it's not a good convention to include a mutable field in `equals()` or `hashCode()` because they can be used as a key of a `HashMap` or `HashSet`. I would like to suggest to make `lineSplittable` immutable, but add a new method `withLineSplittable(boolean lineSplittable)` instead of `setLineSplittable()` which creates a new `JsonInputFormat` instance with the given value.

Reply:
Good idea!